wheel/generate_type_stubs.py

```diff
@@ -368,6 +368,7 @@ def len(self) -> int: ...
     def aggregate_verify(self, pks: list[G1Element], msgs: list[bytes], sig: G2Element) -> bool: ...
     def items(self) -> list[tuple[bytes, GTElement]]: ...
     def update(self, other: Sequence[tuple[bytes, GTElement]]) -> None: ...
+    def evict(self, pks: list[G1Element], msgs: list[bytes]) -> None: ...
```

Comment:
both `update()` and `items()` use a list of tuples rather than two separate lists for the keys and messages. Is there a good reason to diverge from that for this function?

Reply:
Yes, this follows the expected use case, which is more similar to `aggregate_verify` than `update` and `items` in terms of input data.